src/tabs/tab.component.ts

```diff
@@ -104,7 +104,9 @@ export class Tab implements OnInit {
 	/**
 	 * Set to true to have Tab items cached and not reloaded on tab switching.
 	 */
-	@Input() cacheActive = false;
+	@Input() set cacheActive(shouldCache: boolean) {
+		this._cacheActive = shouldCache;
+	}
```

Comment:
I'm not sure this needs to be a getter/setter?

Reply:
I'm using `tab` in the `tab-header-group` method so it needs the setter to actually trigger the change detection since if `cacheActive` was just an input it would not get changed directly

Reply:
Interesting ... there shouldn't be a difference between setter/getter and properties, but if it's something you've noticed is necessary then I'm fine with it 👍

Reply:
Yeah since its being changed like `this.paneReference.cacheActive = this.cacheActive`, I guess it doesn't count as directly changing the property and for some reason it angular doesn't detect any changes